### PR TITLE
Add GitHub Actions workflows for build and release

### PR DESCRIPTION
**Pull request by NoahAndrews**
Based on #13

### CODE REVIEW

File: src/main/java/com/revrobotics/jni/RevJNIWrapper.java [hunk 1]
```diff
@@ -1,51 +0,0 @@
...
- * POSSIBILITY OF SUCH DAMAGE.
- */
-
-package com.revrobotics.jni;
-
-import java.io.IOException;
-
-import edu.wpi.first.wpiutil.RuntimeLoader;
-
-public class RevJNIWrapper  {
```
Comment (LandryNorris) [inline src/main/java/com/revrobotics/jni/RevJNIWrapper.java:35]: Is this used by the FTC SDK in any way? What did this class do before?

**Review by jfabellera — APPROVED** [review]

File: src/main/java/com/revrobotics/jni/RevJNIWrapper.java (hunk 1 above)
Comment (NoahAndrews) [reply src/main/java/com/revrobotics/jni/RevJNIWrapper.java:35]: At this time, anything CAN related is only for FRC, not FTC.

This project seems to have kind of used the SPARK MAX API (now called REVLib) as a template, and this class came from there. It's never been used as a part of this project.

This project appears to be intended to allow people to control real SPARK MAXes connected via USB from WPILib projects running in simulation mode from their computer. See [this thread](https://github.com/REVrobotics/CANBridge/issues/11) and the [main.cpp](https://github.com/REVrobotics/CANBridge/blob/main/src/test/cpp/main.cpp) file referenced in it. Right now, it's not at all able to do that for Java (and that was never really fleshed out for C++ either), so it makes sense to delete stuff to support Java. We can use REVLib as example code for the Java stuff if we ever finish that original CANBridge vision.

Comment (LandryNorris) [reply src/main/java/com/revrobotics/jni/RevJNIWrapper.java:35]: Got it. Sounds good to delete it then.

Comment (LandryNorris) [reply src/main/java/com/revrobotics/jni/RevJNIWrapper.java:35]: I think having no Java classes does strange things with maven publishing. Should we have a private Stub class to be safe?

**Pull request merged**